README.md

```diff
@@ -5,69 +5,96 @@ Robotic product handling simulation
+
 ### Installation
+- Source the ROS environment script
+	
+	```
+	source /opt/ros/kinetic/setup.bash
+	```
+- Create a "gilbreth_ws" catkin workspace
+ 
+	```
+	mkdir -p ~/gilbreth_ws/src
+	cd ~/gilbreth_ws/
+	catkin init
+	```
+- Fetch the gilbreth source code:
+
+	```
+	cd ~/gilbreth_ws/src
+	wstool init .
+	wstool merge https://raw.githubusercontent.com/swri-robotics/gilbreth/kinetic-devel/gilbreth.rosinstall
+	wstool update
+	rosdep install --from-paths . --ignore-src -y
+	```
+- Build the workspace
 
-Create a catkin workspace.
-```
-http://wiki.ros.org/catkin/Tutorials/create_a_workspace
-```
-Second, go to src folder in your workspace:
-```
-cd ~/catkin_ws/src
-```
-Third, download the Gilbreth and dependent packages:
-```
-wstool init .
-wstool merge https://github.com//gilbreth/kinetic-devel/gilbreth.rosinstall
-wstool update
-rosdep install --from-paths . --ignore-src -y
-```
-Make the files
-```
-cd ~/catkin_ws
-catkin_make
-```
+	```
+	cd ~/gilbreth_ws
+	catkin build
+	```
 
 ## Running Gilbreth
 
 ### Setup
-Source the workspace and add the Gazebo model path environment variables
-```
-cd ~/catkin_ws
-source devel/setup.bash
-bash src/gilbreth/gilbreth_gazebo/scripts/export_gazebo_model_path.bash
-```
+- Source the catkin workspace 
+
+	```
+	cd ~/catkin_ws
+	source devel/setup.bash
+	```
 
 ### View the URDF
-```
-roslaunch urdf_tutorial display.launch model:=`rospack find gilbreth_support`/urdf/gilbreth.xacro
-```
+
+	```
+	roslaunch urdf_tutorial xacrodisplay.launch model:=$(rospack find gilbreth_support)/urdf/gilbreth.xacro
+	```
+    
+   It may be necessary to select "world" in the "Fixed Frame" drop down in the rviz window.
+    
 
 ### Run the simulation environment
 
-Roslaunch the gazebo environment:
-```
-roslaunch gilbreth_gazebo gilbreth.launch
-```
-Activate the gripper:
-```
-rosservice call /gilbreth/gripper/control "enable: <true/false>"
-```
-Activate the conveyor:
-```
-rosservice call /gilbreth/conveyor/control "state: power: <(0.0 - 100.0)>"
-```
-Control the conveyor part spawner:
-```
-rosservice call /[start/stop]_spawn
-```
+1. Launch the gazebo simulation environment:
+
+	```
+  	roscd gilbreth_gazebo
+  	source scripts/env_setup.bash
+	roslaunch gilbreth_gazebo gilbreth.launch rviz:=false
+	```
+
+  	- The **source scripts/env_setup.bash** command sets up environment variables needed
+  	by the gazebo simulator.  
+    - Use "rviz:=true" to show rviz
+  	
+1. Activate the gripper:
+
+	```
+	rosservice call /gilbreth/gripper/control "enable: true"
+	```
+    Use "false" in order to turn off the gripper
+
+1. Activate the conveyor:
+
+	```
+	rosservice call /gilbreth/conveyor/control "state: power: 100.0"
+	```
+    The "power" can range from 0.0 to 100.0.  Use 0.0 to stop the conveyor
+
+1. Start the part spawner:
+	
+	```
+	rosservice call /start_spawn "{}"
+	```
+    This command will make parts appear on the conveyor at random intervals.
```

Comment:
Maybe worth mentioning its configurable via .yaml file?

Reply:
What's configurable?

Reply:
The comment that parts will appear on the conveyor at random seems like the user doesn't have any control over those parts. Maybe it's worth mentioning that those parts and their associated randomized parameters are configurable